climpred/classes.py

```diff
@@ -719,11 +718,6 @@ def compute_uninitialized(
         Returns:
             Result of the comparison as a Dataset.
         """
-        has_dataset(
-            self._datasets['uninitialized'],
-            'uninitialized',
-            'compute an uninitialized metric',
-        )
```

Comment:
Is this true? Can you do the uninitialized computation without the uninitialized dataset there?

Reply:
no. was too fast...